examples/rllib/green_wave.py

```diff
@@ -53,12 +86,12 @@ def get_flow_params(col_num, row_num, additional_net_params):
             departLane='free',
             departSpeed=20)
 
-    net_params = NetParams(
+    net = NetParams(
```

Comment:
Is there a reason we're shortening net_params -> net, and initial_config -> initial?

Reply:
since the same variable names are available in the main() portion of this file, there is a chance, if names are changed, that we will be modifying the main() version of the name. This is just a precaution against that.